common/functions/src/scalars/dates/timeslot.rs

```diff
@@ -47,24 +46,27 @@ impl Function for TimeSlotFunction {
         Ok(false)
     }
 
-    fn eval(&self, columns: &DataColumnsWithField, input_rows: usize) -> Result<DataColumn> {
-        let fmt = "%Y-%m-%d %H:%M:%S";
-        let time_str: DataValue = columns[0].column().clone().to_values()?[0].clone();
-        let parse_result = NaiveDateTime::parse_from_str(time_str.to_string().as_str(), fmt);
-        let date = parse_result.unwrap();
-        let minute = date.minute();
-        let mut new_minute: u32 = 0;
-        if minute >= 30 {
-            new_minute = 30;
+    fn eval(&self, columns: &DataColumnsWithField, _input_rows: usize) -> Result<DataColumn> {
+        let mut timestamp = Utc::now().timestamp_millis() / 1000;
+        let len = columns.len();
+        if len > 0 {
+            let args = columns[0].column().clone().to_values()?;
+            timestamp = args[0].as_i64()?;
```

Comment:
We can't just get one row as `timestamp`.

`timeSlow(DateTime)` function not just return DataColumn::Constant, it returns `DataColumn::Array`.


You can do this via 

```
let timecol = columns[0].column();
let timecol = timecol.u32()?;


timecol.iter()....
```

Reply:
Does this mean there will be a behavior like "timeSlot(DateTime1, DateTime2...)"

Reply:
Not yet, everything is column in functions :

- `timeSlot(x)` x is a column.
- `add(a,b)` a and b are columns.